src/components/navigation/PewPewNavBar.vue

```diff
@@ -145,6 +145,9 @@ export default {
 };
 </script>
 <style lang="postcss">
+a {
+  text-decoration: none !important;
+}
```

Comment:
@pablo014 
This style isn't scoped to `PewPewNavBar` component, therefore, setting `text-decoration: none !important;` will be applied globally. We can't do this because `<a>` tags are used throughout the application.

A better solution would be adding `no-underline` tailwind class to `pewpew__navlink` class.

Reply:
done